### PR TITLE
[res] TensorFlow python examples of conv operator

### PR DESCRIPTION
**Pull request by binarman**
Add examples on ordinary and dilated conv operators

ONE-DCO-1.0-Signed-off-by: Alexander Efimov <a.efimov@samsung.com>

### CODE REVIEW

File: res/TensorFlowPythonExamples/examples/conv_1/__init__.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import tensorflow as tf
```
[the remark this answers is not in the document]
Comment (binarman) [reply res/TensorFlowPythonExamples/examples/conv_1/__init__.py:1]: ok, I'll fix it

File: res/TensorFlowPythonExamples/examples/conv_2/__init__.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import tensorflow as tf
+import numpy as np
+
+in_ = tf.compat.v1.placeholder(tf.float32, shape=(1, 32, 32, 3), name="Hole")
+
+filters = np.array(
+    np.random.uniform(low=-1., high=1, size=[5, 5, 3, 32]), dtype=np.float32)
+strides = (1, 2, 2, 1)
+dilations = (2, 2)
```
[the remark this answers is not in the document]
Comment (binarman) [reply res/TensorFlowPythonExamples/examples/conv_2/__init__.py:9]: Looks like this is a restriction of versions prior 1.15

documentation for 1.15 and above says:
> An int or list of ints that has length 1, 2 or 4, defaults to 1.

I'll change it to more general [4] form

File: res/TensorFlowPythonExamples/examples/conv2d_2/__init__.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import tensorflow as tf
+import numpy as np
+
+in_ = tf.compat.v1.placeholder(tf.float32, shape=(1, 32, 32, 3), name="Hole")
+
+filters = np.array(
+    np.random.uniform(low=-1., high=1, size=[5, 5, 3, 32]), dtype=np.float32)
```
Comment (parjong) [inline res/TensorFlowPythonExamples/examples/conv2d_2/__init__.py:7]: (Optional) You may use ``np.random.uniform(...).astype(np.float32)`` instead ;-)

Comment (seanshpark) [reply res/TensorFlowPythonExamples/examples/conv2d_2/__init__.py:7]: @binarman  , apply suggestion? or merge as-is ?

Comment (binarman) [reply res/TensorFlowPythonExamples/examples/conv2d_2/__init__.py:7]: @parjong
actually you suggestion sound good

Comment (seanshpark) [reply res/TensorFlowPythonExamples/examples/conv2d_2/__init__.py:7]: OK, I'll wait for new changes :)

**Pull request merged**